README.md

```diff
@@ -1,8 +1,7 @@
 # Programmers-Club (PC)
 Programmers Club at CAUP. "Journal club" about programming for astronomers.
 
-The programmer's club will be held at the end of the month on Thursdays from
-11.00 - 12.00.
+The programmer's club first edition will be held at the 3rd of December. Further details will be provided in the (very) near future.
```

Comment:
This isn't the first edition :) Maybe better to write second edition?
```suggestion
The programmer's club second edition will be held at the 3rd of December. Further details will be provided in the (very) near future.
```

All the best with the re-birth of Programmer's Club.

Reply:
Thanks,  I overlooked that small detail.